__tests__/game.test.js

```diff
@@ -0,0 +1,25 @@
+import gameBoard from '../src/dom/game';
+
+// tests empty board
+test('it should return an array of empty strings', () => {
+  const newBoard = gameBoard.board;
+  expect(newBoard).toEqual(['', '', '', '', '', '', '', '', '']);
+});
+
+// tests checkWins function
+test('it returns true for a winning move', () => {
+  const newBoard = ['X', 'X', 'X', '', '', '', '', '', ''];
+  expect(gameBoard.checkWins('X', newBoard)).toEqual(true);
+});
+
+// tests checkDraws function
+test('it returns true for a full board without winning combinations', () => {
+  const newBoard = ['X', 'O', 'X', 'X', 'O', 'O', 'O', 'X', 'X'];
+  expect(gameBoard.checkDraws(newBoard)).toEqual(true);
+});
+
+// tests clearBoard function
+test('it clears the board', () => {
+  const newBoard = ['X', 'O', 'X', 'X', 'O', 'O', 'O', 'X', 'X'];
+  expect(gameBoard.clearBoard(newBoard)).toEqual(['', '', '', '', '', '', '', '', '']);
+});
```

Comment:
It is very common to use `describe` blocks to group related tests together
[ jestjs.io Docs(Describe Block) ](https://jestjs.io/docs/en/api#describename-fn)
[This](https://dev.to/valentinogagliardi/jest-tutorial-for-beginners-getting-started-with-jest-for-javascript-testing-e6c) is a great article to learn more about testing.